chrome/content/zotero/preferences/preferences.xhtml

```diff
@@ -35,7 +35,6 @@
 <?xml-stylesheet href="chrome://zotero/skin/zotero.css"?>
 <?xml-stylesheet href="chrome://zotero-platform/content/overlay.css"?>
 <?xml-stylesheet href="chrome://zotero-platform-version/content/style.css"?>
-<?xml-stylesheet href="chrome://zotero-platform/content/zotero.css"?>
```

Comment:
Hi @AbeJellinek , this removes the styles we apply to form elements (button, menuitem, etc.) on Windows, which I think is necessary. Do you think I should create a PR to include those again?


It's currently:

<img width="589" alt="image" src="https://github.com/user-attachments/assets/67a6b8ea-18c4-40d5-90a6-c0c20a513f00">